test/devel_env.sh

```
@@ -0,0 +1,267 @@
+# ---------------------------------------------------------------------------
+# Start the daemon (-d = don't fork, we background it with &)
+# ---------------------------------------------------------------------------
+"$DEVEL_BASE/bin/lsmd" \
+    --plugindir="$DEVEL_BASE/plugins" \
+    --socketdir="$LSM_UDS_PATH" \
+    --confdir="$DEVEL_BASE/config" \
+    -d -v > "$DEVEL_BASE/logs/lsmd.log" 2>&1 &
+
+sleep 2
+
+DEVEL_LSMD_PID="$(ps aux | grep "[l]smd.*$LSM_UDS_PATH" | awk '{print $2}')"
```

Shellcheck (0.11.0)

[info] Consider using pgrep instead of grepping ps output.

(SC2009)